website/pages/component/[id]/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { GetStaticPaths, GetStaticProps } from 'next';
+import fs from 'fs';
+import path from 'path';
+import util from 'util';
+import { css } from 'emotion';
+import { GridContainer, GridItem } from '../../../components/grid/Grid';
```

Comment:
Maybe not for this PR, but may want to look into https://nextjs.org/docs/advanced-features/module-path-aliases to make these imports less painful.

Reply:
I already have baseURL in my tsconfig so this should just work 🤔

Reply:
Indeed it does :)